tests/test_output_formatter.cpp

```diff
@@ -177,6 +216,11 @@ auto construct_multiple_instances_info_reply()
     info_entry->set_image_release("18.04 LTS");
     info_entry->set_id("ab5191cc172564e7cc0eafd397312a32598823e645279c820f0935393aead509");
 
+    info_entry = info_reply.add_info();
+    info_entry->set_name("smitten-bluegill");
+    info_entry->mutable_instance_status()->set_status(mp::InstanceStatus::SUSPENDED);
+    info_entry->set_image_release("Debian GNU/Linux 10 (buster)");
+
```

Comment:
I added some tests here to exercise the formatters. This specific entry tests this line: https://github.com/canonical/multipass/blob/control-network-interfaces-no-driver-specific-matching/src/client/cli/formatter/table_formatter.cpp#L71
Removing it will decrease coverage by 1 line.

Reply:
But that line isn't touched by this PR, is it? So the diff coverage isn't affected, right? Like, `multipass info` is orthogonal to this, at least for now, isn't it? Sorry if I am missing something.

Note that I am not saying this change isn't worthy. If it improves our testing coverage, it is surely a welcome change :slightly_smiling_face: But I don't see how it relates to this PR and we normally try to keep unrelated changes in separate PRs. 

Sometimes the line is blurry. For instance, sometimes there is code dealing with A, B, and C, then you want to introduce code for D doing something different, and feel like you need to change A, B, C to match; or you need to refactor to avoid duplicated code; or fix something wrong in E and F when you add G; or you need to fix the order of I and H when adding J. I'd say it's better to keep things in the same PR in those cases. Is that somehow the case here?

Reply:
Oh, now I see the point. Right, I didn't add that line in this PR, but when adding the tests just checked coverage of that file, not of the diff. Yeah, makes sense to remove those lines.